orangecontrib/text/tests/test_bag_of_words.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from unittest.mock import patch
+
+from orangecontrib.text.bagofowords import BagOfWords
+from orangecontrib.text.corpus import Corpus
+
+
+def progress_callback():
+    return
+
+
+def error_callback(*args):
+    return
+
+
+def mock_gensim_dictionary():
+    raise Exception
+
+
+class BagOfWordsTests(unittest.TestCase):
+
+    def setUp(self):
+        self.bow = BagOfWords(
+                progress_callback=progress_callback,
```

Comment:
OK. As of now they only increase the coverage, but we do not check for anything. Is there a way to assure that they function properly — i.e. that they are even called and that the number of calls is correct?